src/seg_utils.py

```diff
@@ -363,4 +420,34 @@ def segment(net, device, inference_loader, qlty_object, tiled_client):
                 print(f'Frame {frame_number+1} saved to Tiled')
                 frame_number+=1
 
-    return frame_number
\ No newline at end of file
+    return frame_number
+
+def crop_seg_save(net, device, image, qlty_object, parameters, tiled_client, frame_idx):    
+    assert image.ndim==2
+    # Standardization
+    image = (image - np.mean(image)) / np.std(image)
```

Comment:
When applying standardization, it is necessary to use the same mean and variance for both all sets (training, validation, and testing). My suggestion would be to add the mean and variance to the model metadata so it can be used for inference later.

Reply:
Sounds good. I thought the reason of doing standardization is to bring all data set to be confined with a fixed mean and std. But your point does make sense. Working on that right now.

Reply:
Update and wrap this comment up: right now we are doing pixel / 255 across frames, and I've added a TODO in code for future discussion of normalization.